bliss/predict.py

```diff
@@ -33,7 +58,7 @@ def align(img, ref_wcs):
     for bnd in range(img.shape[0]):
         inputs = (img[bnd], ref_wcs[bnd])
         reproj, footprint = reproject_interp(  # noqa: WPS317
-            inputs, ref_wcs[2], order="bicubic", shape_out=img[bnd].shape
+            inputs, ref_wcs[SDSS.BANDS.index("r")], order="bicubic", shape_out=img[bnd].shape
```

Comment:
Should we make this an attribute in the survey (or reuse `survey.prior_flux_ref_band`) instead? That way we could align to the same band in both SDSS and DECaLS even though the order is different (i.e. 2 is `r` in SDSS, but `i` in DECaLS).

Reply:
Make sense to me. Maybe just `survey.reference_band`? Because it's not just for prior flux, it's also for alignment.

Reply:
Done.